### PR TITLE
Selenium Marketplace app candidate

### PR DESCRIPTION
**Pull request by scndel**
_Monitoring_ tab chosen; maybe _Testing_ tab would be more relevant if another tools of this category would appear.

Successful deployment should show this in browser :
Screenshot 2019-10-30 at 23 41 19

### CODE REVIEW

File: selenium/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: Selenium
+version: 3.141.59
+maintainer: "@scndel"
+description: Selenium is a web browser testing framework. This app deploys a ready hub with Chrome node.
+PLEASE PROVIDE AT LEAST 2 GB MEMORY FOR STABILITY.
```
Comment (andyjeffries) [inline selenium/manifest.yaml]: This needs to be on one line - it isn't valid YAML this way - that's why the tests failed. Thanks.

Comment (alexellis) [reply selenium/manifest.yaml]: Would it be better if it wasn't in all caps?

**Comment by scndel** [comment]
> Is there a helm chart that can be used for this instead?

Yes, like for many popular apps. I thought that here advantage was to have out of the box ready app available in _Marketplace_.

> How come it only ships Chrome and not Firefox?

My own testing shows that Firefox node containers are not reliable with _K3S_. Being in a Beta context suggest me that they're better to avoid them in an iterative approach.

> What authentication is in place?

_Selenium_ is agnostic about authentication, by design. You can of course tackle this issue from different network levels. 
In our case, _Ingress_ grants access by URL, which is of course an anonymous and not secured access from a Sec point of view. I can add it as a warning in post_install.md if you find it relevant.

**Comment by scndel** [comment]
Too strict as a tester

**Pull request merged**